src/components/base/outline-accordion/outline-accordion.ts

```diff
@@ -0,0 +1,116 @@
+import { html, TemplateResult, CSSResultGroup } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import { OutlineElement } from '../outline-element/outline-element';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import componentStyles from './outline-accordion.css.lit';
```

Comment:
This question might be the result of my lack of experience with Outline and Lit, but I'm wondering why in this import line, you added a `.lit` file extension to the CSS file, when the CSS file itself just has a `.css` file extension. Is this a Lit coding convention, which causes some CSS post-processing to be triggered?

Reply:
Yeah this has to do with how we are processing the CSS

Reply:
No that's a good question that I'm not fully clear on myself. When we build SB/run yarn start we call ./scripts/lit-styles.js which generates a "standard" css (**.css.lit) file from the tailwind styles in the **.css file. So we're telling the component to look into that generated file for the styles. The why of all of this is a question better directed to @himerus.